File: Backend/twetter/src/main/java/com/bartdebever/twetter/services/RoleService.java
pacvoid addFollow(int followerId, int followingId)kage com.bartdebever.twetter.services;

import com.bartdebever.twetter.models.Role;
import com.bartdebever.twetter.services.interfaces.IRoleService;

/**
 * A service to perform CRUD actions for the Role class.
 */
public class RoleService extends CrudService<Role> implements IRoleService {
}

File: Backend/src/main/kotlin/services/RoleService.kt
package services

import models.Role
import services.interfaces.IRoleService
import javax.enterprise.context.RequestScoped

/**
 * A service to perform CRUD actions for the Role class.
 */
@RequestScoped
open class RoleService : CrudService<Role>(), IRoleService {

    override val all: List<Role>?
        get() = null

    override fun getById(id: Int): Role {
        return entityManager!!.find(Role::class.java, id)
    }
}
